node/src/main/scala/org/bitcoins/node/PeerFinder.scala

```diff
@@ -166,7 +167,7 @@ case class PeerFinder(
     //delete try queue
     _peersToTry.clear()
 
-    _peerData.foreach(_._2.client.close())
+    _peerData.foreach(_._2.client.foreach(_.close()))
```

Comment:
This is not a good idea. This is dropping a `Future` on the floor here. 

Don't use `Future.foreach()`. If an error occurs in the future it will be swallowed and not propgated upwards.